packages/sampler/stories/qa-stories/Scroller.js

```diff
@@ -81,4 +83,82 @@ storiesOf('Scroller', module)
 				</div>
 			</Scroller>
 		)
-	);
+	)
+	.add(
+		'With Many ExpandableList',
+		() => (
+			<Scroller
+				direction={select('direction', prop.direction, Scroller, 'vertical')}
+				focusableScrollbar={boolean('focusableScrollbar', Scroller, false)}
```

Comment:
Are these extra knobs needed for this test?  What about the action loggers?  It's OK to have them, but if they won't be used in these tests, maybe it'll be less confusing to not have them?

Reply:
I was thinking about it too....I can remove them if it is okay.

Reply:
removed action loggers and horizontalScrollbar.  Kept focusableScrollbar.